src/attributes.jl

```diff
@@ -964,7 +964,7 @@ struct VariableName <: AbstractVariableAttribute end
 """
     VariablePrimalStart()
 
-A variable attribute for the initial assignment to some primal variable's value that the optimizer may use to warm-start the solve.
+A variable attribute for the initial assignment to some primal variable's value that the optimizer may use to warm-start the solve. May be a number or nothing (unset).
```

Comment:
nothing -> `nothing`

Reply:
Already fixed in my local copy. Will be together with the fix of the other problems.